_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -216,7 +216,14 @@ public static function register_endpoints() {
 			'permission_callback' => __CLASS__ . '::manage_modules_permission_check',
 		) );
 
-		// Jumpstart
+		// Return current Jumpstart status
+		register_rest_route( 'jetpack/v4', '/jumpstart', array(
+			'methods' => WP_REST_Server::READABLE,
+			'callback' => __CLASS__ . '::jumpstart_status',
+			'permission_callback' => __CLASS__ . '::update_settings_permission_check',
```

Comment:
Suggest aligning this block on the `=>` using spaces - this is the pattern used throughout WP core, and it makes the code much easier to read.